packages/components/src/alignment-matrix-control/utils.tsx

```diff
@@ -31,16 +35,28 @@ export const ALIGNMENT_LABEL: Record< AlignmentMatrixControlValue, string > = {
 export const ALIGNMENTS = GRID.flat();
 
 /**
- * Parses and transforms an incoming value to better match the alignment values
+ * Normalizes an incoming value to better match the alignment values
+ *
+ * @param value an alignment value to normalize
+ *
+ * @return The normalized value
+ */
+export function normalizeValue( value: VoidableAlignmentMatrixControlValue ) {
```

Comment:
I even wonder if the extra function here is just unnecessary abstraction. Perhaps we can just inline that into `transformValue`?

This will solve another problem I have with having a separate `normalize` and `transform` functions. What do we really transform? One has to read the entire function to understand. So I'd instead combine them into one, and likely name them `normalizeValue` like the one here.

Reply:
At one point `normalizeValue` was being used independently outside of `utils`, so needed to be separate, but a bit of a refactor made that no longer true.

Given that neither should now an exported function, for me there's also an argument to keep them separate but not call `normalizeValue` from `transformValue`, given their usage later.

Perhaps premature/irrelevant optimisation, but for example...

```js
function normalizeValue (value) {
  return (value === 'x' ? 'x x' : value)?.replace('-', ' ');
}

const valueId = normalizeValue(value)?.replace(' ', '-');
```

This is potentially doing some unnecessary string replacement that two separate functions would avoid. But I'm not super tied to this issue, given the volumes that we're operating at!

Will combine and worry about performance gains when we need them.